### ISSUE TITLE
Weird mask preprocessing even without any resizing

### ISSUE THREAD

**Issue by JacobTempleton** [milesial/Pytorch-UNet#387]
When ground-truth masks are processed holes in the mask develop around the edges (but not the actual edge itself). This behavior occurs even when scaling is set to 1 and the nearest-neighbor mask resampling in @preprocessing is skipped. I am using the DUTS-TR data-set running training on CPU.

Original Mask:
![821original](https://user-images.githubusercontent.com/94549669/187040529-95e8d198-940b-4594-97e3-2614c07cd8cb.png)

Processed Mask:
![821model](https://user-images.githubusercontent.com/94549669/187040541-f7a9b35c-2516-4726-a950-d46fbab5504d.png)
